proxy/test_helpers.go

```diff
@@ -3,6 +3,8 @@ package proxy
 import (
 	"context"
 	"fmt"
+	"github.com/coinbase/redisbetween/utils"
+	"github.com/google/uuid"
```

Comment:
Nit: Pull down `github.com` imports to separate group

Reply:
done